clicache/integration-test/ThinClientCSTXN.cs

```diff
@@ -409,7 +409,9 @@ public void SuspendResumeCommit()
         resumeEx = true;
       }
       Assert.AreEqual(resumeEx, true, "The transaction should not be resumed");
-      Assert.AreEqual(CacheHelper.CSTXManager.Suspend(), null, "The transaction should not be suspended");
+
+      // The transaction should not be suspended
```

Comment:
Was your intention to add a test for the exception here?

Reply:
Yup, thanks for catching.